style.css

```diff
@@ -0,0 +1,81 @@
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.corner__geometry {
+  height: 111px;
+  width: 74px;
+  margin: 0;
+}
+
+.nav__icon {
+  width: 20px;
+  height: 2px;
+}
+
+.headline {
+  display: flex;
+  justify-content: left;
+}
+
+.text__article {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+}
+
+.text {
+  margin-left: 30px;
+}
+
+.main__article {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.title {
+  height: 210px;
+  width: 80%;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 80px;
+  color: #1a2236;
+  margin: 0;
+}
+
+.bio {
+  width: 60%;
+  height: 84px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+}
```

Comment:
As requested by the previous reviewer, kindly use the same font as the one displayed on Figma 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/198796588-06a88bc1-d9ba-4833-a2c0-3e35b3351f14.png)  | ![image](https://user-images.githubusercontent.com/42799579/198796465-33771f43-05c9-4469-a997-d26c45243fd2.png)  |